bgbb/sql/sql_utils_tests.py

```diff
@@ -0,0 +1,44 @@
+import pandas as pd
+import datetime as dt
+
+
```

Comment:
This module would be better framed as fixture tests instead of runtime assertions.